flight/PiOS/STM32F4xx/inc/pios_dmashot.h

```diff
@@ -0,0 +1,78 @@
+#ifndef PIOS_DMASHOT_H
+#define PIOS_DMASHOT_H
+
+#include <stdint.h>
+
+#include <pios_stm32.h>
+#include "stm32f4xx_tim.h"
+
+#include "pios_tim_priv.h"
+
+
+#define DMASHOT_MAX_BUFFERS						8
+
+#define DMASHOT_MESSAGE_WIDTH					16
+#define DMASHOT_MESSAGE_PAUSE					2
+#define DMASHOT_STM32_BUFFER					(DMASHOT_MESSAGE_WIDTH+DMASHOT_MESSAGE_PAUSE)
+
+#ifdef STM32F446xx
+
+#define DSHOT150_FREQUENCY						11250000
```

Comment:
Most of these belong in a private header, or the implementation (.c), not a public header (public headers should be restricted to minimal public interface only).